code/modules/examine/descriptions/armor.dm

```diff
@@ -20,6 +20,28 @@
 			if(100)
 				return "You would be immune to [descriptive_attack_type] if you wore this."
 
+/obj/item/clothing/proc/describe_slowdown()
+	switch(slowdown)
+		if(-1)
+			return "It looks like it might actually make you faster!"
```

Comment:
Couldnt you do something like (-INFINITY to 0) after the check for 0, in order to have it cover all speed up levels, not just specifically -1?

Reply:
Done, although there's only one other case where armour speeds you up and that's the commonwealth medical hardsuit (-0.5).

Reply:
Future proofing is always beneficial.